src/ValidationContext.php

```diff
@@ -243,16 +243,11 @@ public function getAttributeLabel(): ?string
     /**
      * Get translated attribute name.
      *
-     * @return string|null Translated attribute name. `null` if a single value is validated and there is nothing
-     * to translate.
+     * @return string Translated attribute name. `value` if a single value is validated and a label is not set.
      */
-    public function getTranslatedAttribute(): ?string
+    public function getTranslatedAttribute(): string
     {
-        if ($this->attribute === null) {
-            return null;
-        }
-
-        $label = $this->attributeLabel ?? $this->attribute;
+        $label = $this->attributeLabel ?? $this->attribute ?? 'value';
```

Comment:
Maybe make `value` customizable as well?

Reply:
In what case may this be required?

Reply:
It could be useful to customize text for "value" globally (for every error message). This can be completely another word or the same word with adjective added for example, something like "This value" and similar.